code/__DEFINES/economy.dm

```diff
@@ -1,4 +1,5 @@
 #define STARTING_PAYCHECKS 7
+#define MAX_MAIL_PER_FIVE_MINUTES 15
```

Comment:
Where does this "five minutes" come from? I can't find it in the relevant code, and I think it should be a define as well.

Reply:
That's the fire rate on the economy SS.

Reply:
That's pretty bad, that means that changing that wait has more consequences, when really nothing else should change at all.

Change this to per minute, and utilize delta time and what not.